### PR TITLE
 JavaScript-Capstone Pull Request

### PR DESCRIPTION
**Pull request by alicemirigo92**
1. Use JavaScript to make websites dynamic and build basic single page apps.
2. Use ES6 syntax.
3. Use ES6 modules.
4. Use callbacks and promises.
5. Use webpack.
6. Apply JavaScript best practices and language style guides in code.
7. Use AAA pattern for unit tests.
8. Write units tests for a JavaScript app.
9. Follow Gitflow.
10. Solve simple git conflicts.
11. Send and receive data from an API.
12. Use API documentation.
13. Understand and use JSON.
14. Make JavaScript code asynchronous.
15. Perform a code review for a team member.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,198 @@
...
+> **[Write units tests for a JavaScript app]**
+> **[Send and receive data from an API]**
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Loom Video](https://loom.com/share/e2f473991f3f4cc78aad4c24127a1ab5)
+- [Live Demo Link]()
```
Comment (ichala) [inline README.md]: - [x] Since you don't have a live link yet , kindly remove `Live Demo Link` from your readme file to make it looks professional 🙏

File: src/modules/displayComments.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import commentsCounter from './commentsCounter.js';
+import fetchComment from './fetchcomments.js';
+
+const displayComments = async (id) => {
+  const comments = await fetchComment(id);
+  console.log(comments);
```
Comment (ichala) [inline src/modules/displayComments.js]: - [x] as for the [ JS BEST PRACTICES](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md) , please do not commit console.log to your repo. It’s ugly, it kills performance and it can make confidential data be visible to anyone using the browser tools to look at your website. kindly remove it 🙏

File: test/commentsCounter.test.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+
+describe('commentsCounter,', () => {
+  document.body.innerHTML = '<span id="c-total"></span>';
+  const count = document.getElementById('c-total');
+  const comments = [{ username: 's', comment: 'd', creation_date: '2023-02-17' }];
+  commentsCounter(comments);
+  test('displays the number of comments', () => {
+    expect(count.textContent).toBe('1');
+  });
+});
```
Comment (ichala) [inline test/commentsCounter.test.js]: - [x] Great Job Creating tests for both like & comment counters, but as part of the requirements you need to test all the edge cases as well, kindly add at least 1 edge case example : `the scenario in which there are 0 items on the page (counter function should return 0)`
kindly add an edge case for both Counters Comments & Like 🙏

**Review by ichala — CHANGES_REQUESTED** [review]
Hi Student
Good job so far! 
👏👏👏 
You still need to work on some issues to go to the next project, but you're almost there!

the-simpsons-homer-simpson

## To Highlight 🎉

- No Linter errors  ✔️ 
- Good Design ✔️
- App Works as expected ✔️


 ## Required Changes ♻️


- [ ] **_🛑  Check The Comments Below_**





## Optional suggestions

- N/A

**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by rustam-tolipov — APPROVED** [review]
Hi Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Developer-512px-10

## Highlights
1. No Linters errors 💯
2. All requirements are complete 💯

## Optional suggestions
No optional suggestions :)

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
